Classes/Controller/DummyImageController.php

```diff
@@ -45,15 +45,36 @@ class DummyImageController extends ActionController
      */
     protected $logger;
 
+    /**
+     * @Flow\InjectConfiguration
+     *
+     * @var array
+     */
+    protected $settings;
+
+    /*
+     * Override the default Imagine driver from Neos.Imagine
+     * with a static Imagick instance,
+     * because Vips does not yet support draw
+     */
+    public function initializeObject()
+    {
+        if (isset($this->settings['overrideImagineDriver']) && $this->settings['overrideImagineDriver'] !== false) {
+            $className = 'Imagine\\'.$this->settings['overrideImagineDriver']
+                .'\\Imagine';
+            $this->imagineService = new $className();
+        }
+    }
+
     /**
      * Get a dummy-image.
      *
-     * @param int    $w
-     * @param int    $h
-     * @param string $bg
-     * @param string $fg
-     * @param string $t
-     * @param string $f
+     * @param int         $w
```

Comment:
@gradinarufelix Reformatting should not be part of this PR

Reply:
War automatisch von der IDE